packages/layer-composer/src/generators/context/context-layers.ts

```diff
@@ -0,0 +1,61 @@
+import type { Layer } from 'mapbox-gl'
+import { Group } from '../../types'
+import { ContextLayerType } from '../types'
+
+const settledBoundaries = [
+  '200 NM',
+  'Treaty',
+  'Median line',
+  'Joint regime',
+  'Connection Line',
+  'Unilateral claim (undisputed)',
+]
+
+const CONTEXT_LAYERS: Record<ContextLayerType, Layer[]> = {
+  'eez-areas': [
+    {
+      id: 'eez-base',
+      type: 'fill',
+      paint: {
+        'fill-color': 'transparent',
+        'fill-outline-color': 'transparent',
+      },
+      layout: {},
+      metadata: {
+        interactive: true,
+        group: Group.OutlinePolygonsBackground,
+      },
+    },
+  ],
+  'eez-boundaries': [
+    {
+      id: 'eez_rest_lines',
+      type: 'line',
+      filter: ['match', ['get', 'line_type'], settledBoundaries, true, false],
+      layout: {},
+      paint: {
+        'line-color': '#93c96c',
```

Comment:
Won't that value be overriden by `config.color` ?

Reply:
It will, but there are some apps where we can't customize the color (for example, the carrier portal), and I think it is a good idea to have a color to render the layer equally in every app by default

Reply:
Gotcha... But the thing is, if I understand correctly, this value will never be picked up because you have a fallback <a href="https://github.com/GlobalFishingWatch/frontend/pull/194/files/682c55c93ee349ae58e40db99a141eda316868e8#diff-f5eb444a2dc0a126b3542d6aa5e1d2eecf166cc0d21f51b6d2b677c2a68cddf0R16">here</a> if config.color is not set

Reply:
[Fixed](https://github.com/GlobalFishingWatch/frontend/pull/194/commits/4cd4400ddbd616f98c8ea1182291a9fd48c5b9be) thanks!